src/ParallelAlgorithms/Initialization/AddComputeTags.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <tuple>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "ParallelAlgorithms/Initialization/MergeIntoDataBox.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace Parallel {
+template <typename Metavariables>
+class ConstGlobalCache;
+}  // namespace Parallel
+/// \endcond
+
+namespace Initialization {
+namespace Actions {
+template <typename ComputeTagsList>
```

Comment:
Could you please write dox for this following the format that other actions use? It looks like the other action in this directory is in the `ActionsGroup`.

Reply:
so should this header go in `ParallelAlgorithms/Initialization/Actions` then?

Reply:
hmm yea that seems most consistent

Reply:
Changed the location and added some dox. Let me know if something's wrong or if there are better words for it.